playground/android/build.gradle

```diff
@@ -3,10 +3,14 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
```

Comment:
This can be removed, `google()` is enough

Reply:
Done.